guides/v2.1/frontend-dev-guide/layouts/xml-instructions.md

```diff
@@ -102,7 +102,7 @@ Blocks employ templates to generate {% glossarytooltip a2aff425-07dd-4bd6-9671-2
 
 To pass parameters use the <a href="#argument">`<argument></argument>`</a> instruction. 
 
-<h3 id="fedg_layout_xml-instruc_ex_cont">&lt;container&gt;</h3>
+### &lt;container&gt; {#fedg_layout_xml-instruc_ex_cont}
```

Comment:
Ideally, we would want these `&lt;` to be removed and replaced with the proper symbol for easier reading, but its not imperative for this PR. Keep this in mind for future revisions, however.